test/play_scraper_test.py

```diff
@@ -1,28 +1,37 @@
 from pathlib import Path
+from urllib.error import HTTPError, URLError
 import numpy as np
+import pytest
+from src import PlayScrapingError, PlayPageScraper
 
 ICON_HTML = """
-<img src="file:%s/icon_com.example" class="T75of sHb2Xb">
+<img src="%s" class="T75of sHb2Xb">
 """
 
 CATEGORY_HTML = """
 <a itemprop="genre">Example</a>
 """
 
+SIMPLE_HTML = """
+<p>Simple paragraph.</p>
+"""
+
+ICON_SUBDIR = Path("icon_subdir")
+
 
 class TestPlayPageScraper:
     def test_get_icon(self, test_dir, icon_dir, play_scraper):
         rand_array = np.random.randint(0, 255, 10)
```

Comment:
again -> do we really need rand here -> let's just have a simple hardcoded array to improve visibility and drop dependency on np for the test.